app/assets/stylesheets/application.css

```diff
@@ -5,3 +5,10 @@
  *= require application/safecast
  *= require bootstrap-datetimepicker.min
 */
+
+span {
```

Comment:
Having style on the entire tag might be too risky because it can override other styles. let's make it a CSS class instead like we do [here](https://github.com/Safecast/safecastapi/pull/711/files#diff-4ebb75a2f2d31b2591e792ee7b724436R25)